### ISSUE TITLE
image missing alt text and no <picture>, <source> in index

### ISSUE THREAD

**Issue by Seijno** [driezie/user-experience-enhanced-website#5]
https://github.com/driezie/user-experience-enhanced-website/blob/79ff89ab5ad2d3bdeface9adf7a2568cc0aeeeb9/views/index.ejs#L74

Hier heb je zoals in de titel geen alt, picture of source attributen. Dit doe je wel in andere pagina's dus dit is een kleine fix voor jou!
